shared/modules/selectors/networks.ts

```diff
@@ -65,10 +65,26 @@ export type EvmAndMultichainNetworkConfigurationsWithCaipChainId = (
   caipChainId: CaipChainId;
 };
 
-export const getNetworkConfigurationsByChainId = createDeepEqualSelector(
-  (state: NetworkConfigurationsByChainIdState) =>
-    state.metamask.networkConfigurationsByChainId,
-  (networkConfigurationsByChainId) => networkConfigurationsByChainId,
+export const getNetworkConfigurationsByChainId = (
```

Comment:
There is no reason this needed to use `createDeepEqualSelector`. This already always returns the same output given the same input.